cardpresent/src/main/java/com/fattmerchant/omni/usecase/InitializeDrivers.kt

```diff
@@ -17,6 +17,7 @@ internal class InitializeDrivers(
     class InitializeDriversException(message: String? = null) : OmniException("Could not initialize drivers", message) {
         companion object {
             val NoMobileReadersFound = InitializeDriversException("Couldn't find any mobile readers")
+            val InvalidReaderCredentials = InitializeDriversException("Your account has invalid mobile reader credentials")
```

Comment:
instead of hard coding string here.. we can put them on string.xml and call from each call.. this is because of localisation.